src/main/java/com/algorand/sdkutils/listeners/ResponseGenerator.java

```diff
@@ -435,7 +436,7 @@ public void onEvent(Publisher.Events event, TypeDef type) {
         if (event == Publisher.Events.NEW_PROPERTY) {
             activeList.add(type);
         } else {
-            logger.info("unhandled event: %s", event);
+            logger.info("unhandled event: " + event);
```

Comment:
Really doesn't matter at all, since we don't care about the efficiency of a string concatenation here, but I think `{}` is the correct syntax for these placeholders.

a.k.a the "Log4J" vulnerability!